lib/ansible/plugins/lookup/aws_ssm.py

```diff
@@ -85,7 +85,7 @@
   ignore_errors: true
 
 - name: show fact default to "access failed" if we don't have access
-  debug: msg="{{ "the secret was:" ~ temp_secret | default('couldn\'t access secret') }}"
+  debug: msg="{{ "the secret was:" ~ temp_secret | default("couldn't access secret") }}"
```

Comment:
I think we've got double quotes in double quotes
```suggestion
  debug: msg="{{ 'the secret was:' ~ temp_secret | default('couldn't access secret') }}"
```